src/main/java/org/broadinstitute/hellbender/tools/copynumber/denoising/HDF5SVDReadCountPanelOfNormals.java

```diff
@@ -192,11 +201,19 @@ public SAMSequenceDictionary getSequenceDictionary() {
 
     @Override
     public double[] getSingularValues() {
+        if (getNumEigensamples() == 0) {
+            throw new UnsupportedOperationException("No singular values were available.  " +
```

Comment:
This is opinion, but I would avoid this kind of exception if at all possible (get these sometimes when using `Collections` and it can be a real bummer). You could have this return an empty array `new double[0]` instead and move this logic to the client code.